index.html

```diff
@@ -112,7 +112,9 @@ <h3 class="font-bold">#1</h3>
             <div class="flex justify-end w-full main-content">
               <i class="fa fa-times mr-1 mt-1 x-icon"></i>
             </div>
-            <h2 class="pb-4 text-xl font-semibold main-content">Stats</h2>
+            <h2 class="pb-4 text-xl font-semibold main-content stats-heading">
+              Stats
+            </h2>
```

Comment:
It would be better to have this as a button since it semantically makes more sense that you click on the button to display the stats. Also I can't add the comment but while it does work to have a `div` and mimic a modal, it should be easier to have a `dialog` element instead. If you can I would recommend using the `dialog` element below (lines 118-134 div) and then use the `showModal` and `close` methods to display and hide the modal.